homeassistant/components/zestimate/sensor.py

```diff
@@ -38,10 +37,7 @@
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
 })
 
-
-# Return cached results if last scan was less then this time ago.
-MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=30)
-
+SCAN_INTERVAL = timedelta(hours=24)
```

Comment:
This appears to be unrelated to the referenced issue.  Please don't combine unrelated changes in bug fixes PRs.

Reply:
I've updated the breaking changes description to include more details.  Let me know if you think more explanation is needed.  

Would you like me to break out the SCAN_INTERVAL change in to another PR?